validate/validate.go

```diff
@@ -256,28 +256,30 @@ func (v *Validator) CheckProcess() (msgs []string) {
 		}
 	}
 
-	for _, capability := range process.Capabilities {
-		if err := CapValid(capability, v.HostSpecific); err != nil {
-			msgs = append(msgs, fmt.Sprintf("capability %q is not valid, man capabilities(7)", capability))
+	if v.spec.Platform.OS == "linux" {
+		for _, capability := range process.Capabilities {
```

Comment:
So we should add checks for capabilities on other OSes.  Or move the Linux-specific cap validation into a `CheckLinuxCapabilities` and raise not-implemented-yet errors if someone tries to validate a non-Linux config.  But silently ignoring `process.capabilities` in non-Linux configs will give users a false sense of security.